lib/cache.js

```
@@ -47,6 +47,7 @@ function cache(cache_provider, cache_options)
 				{
 					// If there was an error interacting with the cache, then
 					// fall through to the original function.
+					// (this will never happen in current code state)
```

I'm not sure why I swallowed the error in the redis cache provider. That seems like an oversight that we may want to fix.

However, a user might bring his own cache provider which does emit errors, so this comment isn't quite right.